packages/ui/src/components/ValidatorList/KeystoresDataGrid.tsx

```diff
@@ -84,6 +192,36 @@ export default function KeystoresDataGrid({
       headerClassName: "tableHeader",
       width: 360,
     },
+     // Only render Smooth column if mevSpFeeRecipient is not null (mainnet or prater)
+    ...(mevSpFeeRecipient != null && (network === "mainnet" || network ==="prater") ? [{
+      field: "spSubscription",
+      headerName: "Smooth",
+      description:
+        "Dappnode's Smooth subscription status. Smooth states can take up to 40 minutes to update.",
+      disableReorder: true,
+      disableColumnMenu: true,
+      disableExport: true,
+      sortable: false,
+      align: "center"  as GridAlignment,
+      headerAlign: "center"  as GridAlignment,
+      headerClassName: "tableHeader",
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      renderCell: (rowData: { row: any; }) => {
+        // only render smooth status if tag is "solo"
+        if (rowData.row.tag === "solo") {
+          return (
+            <SmoothStatus
+              rowData={rowData}
+              subscriptionStatus={validatorsSubscriptionMap[rowData.row.pubkey]}
+              mevSpFeeRecipient={mevSpFeeRecipient}
+              oracleCallError={oracleCallError}
+            />
+          );
+        } else {
+          return <span>-</span>;
```

Comment:
You could try to find an emoji like :no_entry_sign: or something similar that lets the user know it is not available for that protocol

Reply:
I have this emoji as "banned" status, didnt want to spam the column too much